grafana/risingwave-dashboard.dashboard.py

```diff
@@ -103,7 +103,7 @@ def timeseries_count(self,
                          title,
                          description,
                          targets,
-                         legendCols=["max"]):
+                         legendCols=["mean"]):
```

Comment:
Shall we show by setting legendCols=["max", "mean"])?

Reply:
I considered this before, but that made the legends too wide (even wider than the graph)

<img width="918" alt="image" src="https://user-images.githubusercontent.com/10192522/211236587-2a6ef769-7529-4156-b064-6cb661a47a8f.png">

<img width="921" alt="image" src="https://user-images.githubusercontent.com/10192522/211237455-54f63bbf-d022-424f-902d-45ae749c251e.png">

Reply:
@hzxa21 By the way, do you think max() is better for latency?

Reply:
By the way*2, an alternative option is use `[]` (no metrics in the legend)

Reply:
> For latency, I think it's better to calculate percentiles in separate metrics, so I think using mean is fine.

+1